__tests__/app.test.js

```diff
@@ -46,3 +46,34 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("should respond with 200 and array of all articles", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((results) => {
+        expect(results.body.articles.length).toBe(13);
+        results.body.articles.forEach((article) => {
+          expect(article).not.toHaveProperty("body");
+          expect(article).toHaveProperty("title");
+          expect(article).toHaveProperty("article_id");
+          expect(article).toHaveProperty("topic");
+          expect(article).toHaveProperty("created_at");
+          expect(article).toHaveProperty("article_img_url");
+          expect(article).toHaveProperty("comment_count");
+          expect(article).toHaveProperty("author");
+        });
+      });
+  });
+  describe("should handle errors", () => {
+    test("should return 400 invalid path", () => {
```

Comment:
This test is wrong and not needed. It's wrong as 400 is bad requests. It's not needed as we don't have a bad request to consider for this endpoint. We don't accept queries on this endpoint so the only thing that can really go wrong is a non-existent 404 which should have been handled previously

Reply:
We should test that the articles return sorted and ordered as expect i.e. created_at, descending